samples/Android/app/build.gradle

```diff
@@ -35,8 +35,8 @@ android {
             java.srcDir '../../Utils/CommandLineUtils'
             java.srcDir '../../Mqtt5/PubSub/src/main/java'
             java.srcDir '../../BasicPubSub/src/main/java'
-            java.srcDir '../../Jobs/src/main/java'
-            java.srcDir '../../Shadow/src/main/java'
+            java.srcDir '../../deprecated/Jobs/src/main/java'
```

Comment:
These being deprecated, should we simply remove them entirely from the Android sample app so people don't use them?